### ISSUE TITLE
[Bug] Scenario languages not shown correctly because of missing data in .ini file

### ISSUE THREAD

**Issue by Quantumrunner** [NPBruce/valkyrie#1485]
### Describe the bug
After adding the l`ocalization.spanish.txt` to one of my scenarios and adding the line to the `quest.ini` the translations are not being added to the `.ini` file when creating the package. All other translations are missing as well.

When opening the mission the translation works fine. Only the entries for the ini files are missing for all languages

When I switch the  valkyrie  UI language to german it also adds german data to ini:
```
name.English=City of the Dead (V1.3)
name.German=Die Stadt der Toten (V1.3)
synopsys.English=When you return to your hometown Vynelvale you find it under siege by undead forces.
synopsys.German=Als ihr in eure Heimatstadt Vynelvale zurückkehrt, findet ihr sie unter Belagerung durch untote Horden.
```

Same for valkyrie  UI  spanish:
```
name.English=City of the Dead (V1.3)
name.Spanish=La ciudad de los muertos (V1.3)
synopsys.English=When you return to your hometown Vynelvale you find it under siege by undead forces.
synopsys.Spanish=Cuando regresáis a vuestra ciudad natal, Vynelvale, descubrís que está siendo asediada por huestes de no muertos.
```
 
But the other language is always missing

### Demo scenario to reproduce
[TranslationError.zip](https://github.com/NPBruce/valkyrie/files/6292217/TranslationError.zip)

### Valkyrie Version
2.5.4

### Desktop
 - OS: Windows 10

**Comment by vidimus** [comment]
I downgraded Valkyrie and I was able to create the package without problems. So it's something with the new version.

**Comment by Quantumrunner** [comment]
Confirmed fixed with Valkyrie 2.5.5 version.
